.github/workflows/package.yaml

```diff
@@ -46,6 +47,14 @@ jobs:
         persist-credentials: false
         path: rancher-desktop-docker-cli
 
+    - name: Set Docker CLI tag from workflow file
+      if: github.event_name == 'pull_request'
+      run: >
+        awk '/default: +v/ {print "DOCKER_CLI_REF=" $2}'
+        "${GITHUB_WORKSPACE}/rancher-desktop-docker-cli/.github/workflows/package.yaml"
+        >>"${GITHUB_ENV}"
+      working-directory: .
```

Comment:
I guess this undoes the default? (No need to reply if this is correct.)

Reply:
I thought the input defaults would only be set when you run the workflow via `workflow_dispatch` and not via `pull_request`. If they are always set, then this step is not necessary.

But I realized now that triggering the workflow on pull requests isn't really necessary; not sure anymore why I added it; I've removed it again. 😄